### DISCUSSION TITLE
Can we add a 'Eway' payment gateway?

### DISCUSSION THREAD

**Discussion by gauravjw** [vanilophp/framework#173] [General]
Hi Team,

We want to implement eway payment gateway. Is there any library for vanilo available like Stripe? SO we can install and use as service provider?

Here is the payment gateway doc
https://eway.io/api-v3/

**Comment by fulopattila122** [comment]
There's an incomplete but functional Stripe module for Vanilo 2: https://github.com/vanilophp/stripe
Would completing the Stripe module help?

To develop your Payment Gateway for Vanilo, here's a starter kit: https://github.com/vanilophp/payment-gateway-skeleton

Some clients have hired the Vanilo Team to implement specific gateways.
